<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>numberPlayers</class>
 <widget class="QWidget" name="numberPlayers">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>701</width>
    <height>374</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="title">
     <property name="font">
      <font>
       <pointsize>30</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Left 4 Dead</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="gameNameLabel">
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Ingrese el nombre de la partida</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="inputGameName"/>
   </item>
   <item>
    <widget class="QLabel" name="gameNumberLabel">
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Ingrese la cantidad de jugadores</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
    <item>
    <widget class="QLabel" name="gameCodeLabel">
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string></string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSpinBox" name="inputNumber">
     <property name="maximum">
      <number>10</number>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="2">
    <widget class="QLabel" name="label_10">
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Eliga un modo de juego</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
    <item row="7" column="0" colspan="2">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QFrame" name="frame_4">
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <widget class="QLabel" name="label_11">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>10</y>
            <width>101</width>
            <height>20</height>
           </rect>
          </property>
          <property name="text">
           <string>Clear the zone</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame_5">
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <widget class="QLabel" name="label_12">
          <property name="geometry">
           <rect>
            <x>-4</x>
            <y>10</y>
            <width>101</width>
            <height>20</height>
           </rect>
          </property>
          <property name="text">
           <string>Survival</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QFrame" name="player_1_option_2">
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <widget class="QLabel" name="label_14">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>10</y>
            <width>101</width>
            <height>20</height>
           </rect>
          </property>
          <property name="text">
           <string>Opción 1</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="player_2_option_2">
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <widget class="QLabel" name="label_15">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>10</y>
            <width>101</width>
            <height>20</height>
           </rect>
          </property>
          <property name="text">
           <string>Opción 2</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="8" column="0" colspan="2">
    <widget class="QSpinBox" name="gameMode">
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>2</number>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="leftMargin">
      <number>150</number>
     </property>
     <property name="rightMargin">
      <number>150</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QPushButton" name="sendNumber">
       <property name="text">
        <string>Enviar</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
